packages/mobile/src/store/sagas.ts

```diff
@@ -75,13 +76,13 @@ export default function* rootSaga() {
     ...trackPageSagas(),
     ...collectionPageSagas(),
     ...feedPageSagas(),
-    ...trendingPageSagas(),
     ...exploreCollectionsPageSagas(),
-    ...trendingPageSagas(),
     ...explorePageSagas(),
+    ...trendingPageSagas(),
```

Comment:
ah this might have fixed the trending lineup stuff! i think since there were double the sagas, it was calling extra?